### PR TITLE
On-disk Virtual Merkel Tree

### PR DESCRIPTION
**Pull request by rbair23**
This PR proposes a new HIP for on-disk virtualized merkle trees.

### CODE REVIEW

File: HIP/hip-0000-virtual-merkle.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+- hip: <HIP number (this is determined by the HIP editor)>
```
Comment (sergmetelin) [inline HIP/hip-0000-virtual-merkle.md:1]: Please assign HIP-25 to this HIP document and rename it to `hip-25.md`

File: HIP/hip-0000-virtual-merkle.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+- title: Virtual Merkle
+- author: Richard Bair (@rbair23), Jasper Potts (@jasperpotts)
+- type: Standards Track
+- category: Core
+- status: Draft
+- created: 2021-09-07
+- discussions-to: <a URL pointing to the official discussion thread>
```
Comment (sergmetelin) [inline HIP/hip-0000-virtual-merkle.md:8]: Please link to this Discussion thread: https://github.com/hashgraph/hedera-improvement-proposal/discussions/139

File: HIP/hip-0000-virtual-merkle.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+- updated: <comma separated list of dates>
```
Comment (sergmetelin) [inline HIP/hip-0000-virtual-merkle.md:9]: Please update with the current date

File: HIP/hip-0000-virtual-merkle.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Virtual Merkle
```
Comment (sergmetelin) [inline HIP/hip-0000-virtual-merkle.md:2]: Is the right title `Virtual Merkle` or `Virtual Merkle Tree`?

**Review by sergmetelin — CHANGES_REQUESTED** [review]

File: HIP/hip-0000-virtual-merkle.md (hunk 4 above)
Comment (rbair23) [reply HIP/hip-0000-virtual-merkle.md:2]: I guess a better name is "On-disk Virtual Merkle Tree"

**Review by sergmetelin — APPROVED** [review]
LGTM

**Pull request merged**